src/main/java/core/basesyntax/service/RegistrationService.java

```diff
@@ -1,7 +1,8 @@
 package core.basesyntax.service;
 
+import core.basesyntax.exception.RegistrationException;
 import core.basesyntax.model.User;
 
 public interface RegistrationService {
-    User register(User user);
+    User register(User user) throws RegistrationException;
```

Comment:
Remove the 'throws RegistrationException' from the method signature, as it is not necessary for unchecked exceptions.

Reply:
Done